client/blocks/comment-detail/index.jsx

```diff
@@ -68,8 +69,10 @@ export class CommentDetail extends Component {
 	}
 
 	deleteCommentPermanently = () => {
-		const { commentId, deleteCommentPermanently } = this.props;
-		deleteCommentPermanently( commentId );
+		const { commentId, deleteCommentPermanently, translate } = this.props;
+		if ( isUndefined( window ) || window.confirm( translate( 'Delete this comment permanently?' ) ) ) {
```

Comment:
It actually got the rebase all right, but then I also added a check on the same line of translate. :)